README.md

```diff
@@ -147,12 +147,58 @@ We can also query Shopify using the GraphQL endpoint:
 
 ```python
 token.execute_gql(
-  query,
-  variables,
-  operation_name
+  query: str,
+  variables: Dict[str, Any],
+  operation_name: Optional[str]
 )
 ```
 
+For example, if you want to query a product from shopify you can run:
+
+```python
+query = """
+{
+  shop {
+    name
+  }
```

Comment:
That will give you the Shop Name ... Could be good enough of an example though product query sounds better.